go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/upper/db
```

Comment:
Should let this path contain `"/v3"` according with Go Module conventions for v2+:
- `github.com/upper/db/v3`

Note that this also changes the import path to contain the trailing `v3`, so those paths should be updated to also contain the `/v3` suffix. PS! even the import path changes, does not mean that the _file directory_ path should change.